src/pages/dashboard/dashboard.ts

```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
```

Comment:
Try to keep changes separate if possible, group only related files together when necessary.